lua/sql.lua

```diff
@@ -341,11 +342,15 @@ function sql:insert(tbl, rows)
       s:bind_clear()
       table.insert(ret_vals, s:finalize())
     end
+      local s = self:__parse("SELECT last_insert_rowid()")
+      s:step()
+      last_rowid = s:vals()[1]
+      s:finalize()
```

Comment:
You dont need to select it there is a sqlite3 api function for that.

```c
SQLITE_API sqlite3_int64 sqlite3_last_insert_rowid(sqlite3*);
```
see https://www.sqlite.org/c3ref/last_insert_rowid.html

Reply:
Oh right, had not used ffi before. Should be fixed now with the last commit.